packages/sync/src/replicastore/class-table-checksum.php

```diff
@@ -0,0 +1,386 @@
+<?php
+
+namespace Automattic\Jetpack\Sync\Replicastore;
+
+use Automattic\Jetpack\Sync\Settings;
+use Exception;
+use WP_error;
+
+// TODO add rest endpoints to work with this, hopefully in the same folder
+
+class Table_Checksum {
+	public $table                 = '';
+	public $table_configuration   = array();
+	public $range_field           = '';
+	public $key_fields            = array();
+	public $checksum_fields       = array();
+	public $additional_filter_sql = '';
+
+
+	public $default_tables = array();
+
+	public $salt = '';
+
+	public $allowed_tables = array();
+
+	/**
+	 * Table_Checksum constructor.
+	 *
+	 * @param string $table
+	 * @param string $salt
+	 * @param string $range_field
+	 * @param null   $key_fields
+	 * @param null   $filter_field
+	 * @param array  $checksum_fields
+	 */
+
+	public function __construct( $table, $salt = null ) {
+		$this->salt = $salt;
+
+		global $wpdb;
+
+		$this->default_tables = array(
+			'posts'              => array(
+				'table'           => $wpdb->posts,
+				'range_field'     => 'ID',
+				'key_fields'      => array( 'ID' ),
+				'checksum_fields' => array( 'post_modified' ),
```

Comment:
with post_modified only we don't have a solid check on manual SQL changes. but at the same time those won't get caught by incremental sync either. 

If we wanted to expand I'd add in post_content. also I might think it would be good to go with the _gmt field

Reply:
Updated to use post_modified_gmt in 00762b449

No need to add post_content at this time.